test/Approximations/overapproximate.jl

```diff
@@ -147,6 +147,11 @@ for N in [Float64, Rational{Int}, Float32]
     if N <: AbstractFloat
         @test isequivalent(P, R)
     end
+
+    #overapproximate a SparsePolynomialZonotope with a VPolytope
+    S = SparsePolynomialZonotope(N[-0.5, -0.5], N[1. 1 1 1;1 0 -1 1], zeros(N, 2, 0), [1 0 1 2;0 1 1 0])
+    P = overapproximate(S, VPolytope)
+    @test [N[-1.5, -2.5], N[2.5, -0.5], N[3.5, 0.5], N[-0.5, 2.5], N[-1.5, 1.5]] == vertices_list(P)
```

Comment:
```suggestion
    @test ispermutation([N[-1.5, -2.5], N[2.5, -0.5], N[3.5, 0.5], N[-0.5, 2.5], N[-1.5, 1.5]], vertices_list(P))
```
The vertices do not have to appear in that order for correctness (forgetting that we apply `convex_hull!`). For that reason, we have this helper method `ispermutation`.